visitor-sdk/_methods.md

```diff
@@ -102,10 +102,30 @@ visitorSDK
 
 #### Parameters:
 
-| param    | type   | description    |
-| -------- | ------ | -------------- |
-| file     | file   | File to upload |
-| customId | string | custom file id |
+| param      | type     | description                                                                |
+| ---------- | -------- | -------------------------------------------------------------------------- |
+| file       | blob     | File to upload                                                             |
+| onProgress | function | Callback function - will receive a progress value - number between 0 and 1 |
```

Comment:
Callback function - will receive a progress value - number between 0 and 1 -> Callback function. It will receive a progress value (a number between 0 and 1)